pkg/apis/machineconfiguration.openshift.io/v1/helpers.go

```diff
@@ -20,8 +20,13 @@ func MergeMachineConfigs(configs []*MachineConfig, osImageURL string) *MachineCo
 	}
 	sort.Slice(configs, func(i, j int) bool { return configs[i].Name < configs[j].Name })
 
+	var fips bool
 	outIgn := configs[0].Spec.Config
 	for idx := 1; idx < len(configs); idx++ {
+		// if any of the config has FIPS enabled, it'll be set
```

Comment:
Ah OK I see we went with the last approach.  I think that's sane for this indeed.